src/earn/prepareTransactions.ts

```diff
@@ -164,3 +144,72 @@ export async function prepareWithdrawAndClaimTransactions({
     origin: 'earn-withdraw',
   })
 }
+
+async function prepareWithdrawTransactions({
```

Comment:
this function could use some unit tests, similar to other `prepare<>Transactions` functions

Reply:
Added in bded4c8ca85cb47ce3aea7c82ca1e328e8625458.